app/domain/authentication/authn_jwt/signing_key/fetch_cached_signing_key.rb

```diff
@@ -0,0 +1,45 @@
+module Authentication
+  module AuthnJwt
+    module SigningKey
+      # FetchCachedSigningKey command class is a callable wrapper of FetchSigningKeyInterface interface,
+      # in order to be able to store the signing key in our cache mechanism
+      class FetchCachedSigningKey
+        attr_accessor :authentication_parameters
+
+        def initialize(
+          create_signing_key_factory: CreateSigningKeyFactory.new,
+          logger: Rails.logger
+        )
+          @create_signing_key_factory = create_signing_key_factory
+          @logger = logger
+        end
+
+        def call(authentication_parameters:)
+          @authentication_parameters = authentication_parameters
+          fetch_signing_key_interface
+          fetch_signing_key
+        end
+
+        def key
+          @authentication_parameters.key
```

Comment:
how it is working? i saw the function name is `hash`

Reply:
Removed function key